docs/resources/lifecycle.md

```diff
@@ -33,7 +33,7 @@ resource "octopusdeploy_lifecycle" "example" {
     name                         = "foo"
 
     release_retention_policy {
-      quantity_to_keep    = 0
```

Comment:
I think this could be an issue, I'm assuming 0 means keep all, so 1 is likely a change of behaviour.

Reply:
This is just the docs and I see that the default is set to 30 on the schema, I'm not sure why the docs pick up 1 here. My concern is just that customers copy this example and it's a pretty bad default, 30 or 0 would be preferable.